src/services/AudiusBackend.js

```diff
@@ -1146,11 +1146,11 @@ class AudiusBackend {
       }
 
       if (
-        newMetadata.twitter_handle ||
-        newMetadata.instagram_handle ||
-        newMetadata.tiktok_handle ||
-        newMetadata.website ||
-        newMetadata.donation
+        typeof newMetadata.twitter_handle === 'string' ||
```

Comment:
this seems good to me. I wonder if we should instead do a manual check on empty string and coerce that to null (not sure how the BE would handle, but maybe more semantically correct)

Reply:
I think the BE checks for nulls though, doesn't it? I think empty string is safer?